kernprof.py

```diff
@@ -250,5 +243,6 @@ def strictly_positive(option, opt, value, parser):
         if options.view:
             prof.print_stats(output_unit=options.unit, stripzeros=options.skip_zero)
 
+
 if __name__ == '__main__':
-    sys.exit(main(sys.argv))
+    main()
```

Comment:
Why remove the sys.exit? Doing this will prevent the exit code. 

Consider the following script: 

```python
import sys
import ubelt as ub


def main():
    val = ub.argval('--exit-val', default=None)

    if val == 'None':
        val = None

    if val == 'err':
        raise Exception

    if val is not None:
        val = int(val)
    return val


if __name__ == '__main__':
    mode = ub.argval('--exit-mode', default='sys')
    if mode == 'sys':
        sys.exit(main())
    else:
        main()
```

```bash
Script Commands:
    python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=0 ; \
    echo $? ; \
    python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=1 ; \
    echo $? ; \
    python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=None ; \
    echo $? ; \
    python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=err ; \
    echo $?

    python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=0 ; \
    echo $? ; \
    python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=1 ; \
    echo $? ; \
    python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=None ; \
    echo $? ; \
    python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=err ; \
    echo $?

Results:
    (py38) joncrall@Ooo:~/misc/tests/python$     python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=0 ; \
    >     echo $? ; \
    >     python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=1 ; \
    >     echo $? ; \
    >     python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=None ; \
    >     echo $? ; \
    >     python ~/misc/tests/python/test_exit_code.py --exit-mode=sys  --exit-val=err ; \
    >     echo $?
    0
    1
    0
    Traceback (most recent call last):
      File "/home/joncrall/misc/tests/python/test_exit_code.py", line 46, in <module>
        sys.exit(main())
      File "/home/joncrall/misc/tests/python/test_exit_code.py", line 36, in main
        raise Exception
    Exception
    1
    (py38) joncrall@Ooo:~/misc/tests/python$
    (py38) joncrall@Ooo:~/misc/tests/python$     python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=0 ; \
    >     echo $? ; \
    >     python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=1 ; \
    >     echo $? ; \
    >     python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=None ; \
    >     echo $? ; \
    >     python ~/misc/tests/python/test_exit_code.py --exit-mode=None --exit-val=err ; \
    >     echo $?
    0
    0
    0
```

Reply:
Hey! It's been a long time.

`main()` has no return value so it was actually equivalent to `main(); sys.exit(None)`, so that's why I removed it.